tests/regression_tests/photon_source/test.py

```diff
@@ -0,0 +1,61 @@
+from math import pi
+
+import numpy as np
+import openmc
+
+from tests.testing_harness import PyAPITestHarness
+
+
+class SourceTestHarness(PyAPITestHarness):
+    def _build_inputs(self):
+        mat = openmc.Material()
+        mat.set_density('g/cm3', 0.998207)
+        mat.add_element('H', 0.111894)
+        mat.add_element('O', 0.888106)
+        materials = openmc.Materials([mat])
+        materials.export_to_xml()
+
+        sphere = openmc.Sphere(R=1.0e9, boundary_type='reflective')
+        inside_sphere = openmc.Cell()
+        inside_sphere.region = -sphere
+        inside_sphere.fill = mat
+        root = openmc.Universe()
+        root.add_cell(inside_sphere)
+        geometry = openmc.Geometry(root)
```

Comment:
You can pass `Geometry` a list of cells and it will automatically create a universe for you
```Python
inside_sphere = openmc.Cell()
...
geometry = openmc.Geometry([inside_sphere])
```